bats/tests/helpers/load.bash

```diff
@@ -101,6 +101,7 @@ setup() {
 
 teardown() {
     if [ -z "$BATS_TEST_SKIPPED" ] && [ -z "$BATS_TEST_COMPLETED" ]; then
+        capture_logs
```

Comment:
I don't understand why this is necessary; we capture logs before a factory reset, and in `teardown_file`. Is there any other operation that would delete logs before we can capture them?

Reply:
Yes; starting Rancher Desktop deletes (any existing) logs. So a start/quit/start cycle would lose logs.